dev/com.ibm.ws.messaging.open_jms20_fat/test-applications/JMSProducer/src/jmsproducer/web/JMSProducerServlet.java

```diff
@@ -3702,52 +3729,85 @@ public void testSetTimeToLive_B_SecOff(
         }
     }
 
+    private void displayProducer(JMSProducer jmsProducer, String tag) {
+        System.out.println(tag + ": JMSProducer [ " + jmsProducer + " ]");
+    }
+
+    private void displayMessage(Message message, String tag) throws JMSException {
+        System.out.println(tag + ": Message [ " + message + " ]");
+    }
+
     public void testSetTimeToLive_TCP_SecOff(
         HttpServletRequest request, HttpServletResponse response) throws Throwable {
 
+        String methodName = "testSetTimeToLive_TCP_SecOff";
+
         JMSContext jmsContextQCFTCP = qcfTCP.createContext();
         emptyQueue(qcfTCP, queue1);
         JMSConsumer jmsConsumer = jmsContextQCFTCP.createConsumer(queue1);
         JMSProducer jmsProducer = jmsContextQCFTCP.createProducer();
 
-        TextMessage msgOut = jmsContextQCFTCP.createTextMessage();
+        // jmsProducer.setDeliveryMode(DeliveryMode.NON_PERSISTENT); // Doesn't change the test result.
 
-        long defaultTimeToLive = jmsProducer.getTimeToLive();
-        System.out.println("Default time to live [ " + defaultTimeToLive + " ]");
+        TextMessage msgOut = jmsContextQCFTCP.createTextMessage();
+        displayMessage(msgOut, methodName + " Ougoing Message");
```

Comment:
"Outgoing"?

Reply:
Ah, yeah, that should be "Outgoing"; fortunately, that is just diagnostic output and not impacting the test result.  I'll fix it in a future PR.